src/yb/master/master_tserver.h

```diff
@@ -80,7 +80,9 @@ class MasterTabletServer : public tserver::TabletServerIf,
 
   std::vector<yb::util::WaitStateInfoPtr> GetThreadpoolWaitStates() const override;
 
-  std::vector<WaitStateInfoPB> ActiveUniverseHistory() const override;
+  void ActiveUniverseHistory(tserver::PgActiveUniverseHistoryResponsePB* resp) const override {}
+
+  void GetCQLServerMessenger(tserver::CQLServerMessenger messenger) override {};
```

Comment:
Do we need these changes?

Reply:
CQL server is not accessible from TServer, so the second function installs a callback to get the CQL Server callback. 

The first function, I think that makes it a little more efficient

Reply:
Didn't get it. These calls are in master. Is it because of inheritance that you need them here?

Reply:
Yeah, these are also included in master because of inheritance

Reply:
The base class should have this function rather than all the inheriting classes.

Reply:
The base class has only virtual functions, so there needs to be a no-op definition here. The other examples are similar too